templates/web/makemyisland/report/display.html

```diff
@@ -0,0 +1,78 @@
+[% 
+    SET bodyclass = 'mappage';
+    PROCESS "report/photo-js.html";
+    PROCESS "maps/${map.type}.html";
+
+    problem_title = problem.title_safe _ ' - ' _ loc('Viewing a problem');
+    INCLUDE 'header.html'
+        title = problem_title
+        rss = [ loc('Updates to this problem, FixMyStreet'), "/rss/$problem.id" ]
```

Comment:
FMS ref here. The only changes to this template are turning on share (so I guess that should be a template hook, or could for now add to the IF statement), and doing the FB button a different way - should this be done to the main template?